content/tutorials/manage-the-rippled-server/installation/build-run-rippled-ubuntu.md

```diff
@@ -27,11 +27,49 @@ These instructions use Ubuntu's APT (Advanced Packaging Tool) to install the sof
 
 3. Install dependencies.
 
-        sudo apt-get -y install git cmake pkg-config protobuf-compiler libprotobuf-dev libssl-dev wget
+        sudo apt-get -y install git pkg-config protobuf-compiler libprotobuf-dev libssl-dev wget
 
-4. Compile Boost.
+4. Compile CMake.
 
-    Version 1.1.0 of `rippled` requires Boost version 1.67.0 exactly. Because Boost version 1.67.0 isn't available in the Ubuntu 16.04 software repositories, you must compile it yourself.
+    Version 1.2.0 of `rippled` requires CMake version 3.9.0 or higher. Because CMake version 3.9.0 or higher isn't available in the Ubuntu 16.04 software repositories, you must compile it yourself.
+
+    If you have previously built CMake 3.9.0 or higher for `rippled` and configured the `CMAKE_ROOT` environment variable, you can skip these steps.
+
+      1. Download CMake 3.13.3. For the purposes of this tutorial, we used CMake 3.13.3, which was the latest version available at the time of writing.
+
+              wget https://cmake.org/files/v3.13/cmake-3.13.3.tar.gz
+
+      2. Extract `cmake-3.13.3.tar.gz`.
+
+              tar xvzf cmake-3.13.3.tar.gz
+
+      3. Change to the new `cmake-3.13.3` directory.
+
+              cd cmake-3.13.3
+
+      4. Prepare to build the CMake installer.  ***TODO: stated correctly?***
+
+              ./bootstrap
+
+      5. Build the CMake installer. ***TODO: stated correctly?***
+
+              make
+
+      6. Install CMake.
+
+              sudo make install
+
+      7. Set the environment variable `CMAKE_ROOT` to point to the new `cmake-3.13.3` directory. It's best to put this environment variable in your `.profile`, or equivalent, file for your shell so it's automatically set when you log in. Add the following line to the file:
+
+              export CMAKE_ROOT=/home/my_user/cmake-3.13.3
```

Comment:
I'm not aware of any reason for setting `CMAKE_ROOT`, unless you use it somewhere else in these instructions. Also, if you did a `make install` like above or use the binary installer, it will install in `/usr/local` by default which should be part of the standard search path.